### PR TITLE
Extended WordpressEx to support fetching Users from Wordpress REST

### PR DESCRIPTION
**Pull request by mikehostetler**
Great module! Extended to add support for the Users endpoint.

### CODE REVIEW

File: lib/wordpress_ex/users.ex [hunk 1]
```diff
@@ -0,0 +1,40 @@
+defmodule WordpressEx.Users do
+  @moduledoc """
+  Wordpress User endpoints
+  """
+  import WordpressEx.Client, only: [get: 2]
+  import WordpressEx.StructComposer, only: [compose: 2]
+  alias WordpressEx.Model.User
+
+  @doc """
+  Get a list of categories
```
Comment (tmock12) [inline lib/wordpress_ex/users.ex]:
```suggestion
  Get a list of users
```

I think this was just a copy/paste leftover. Can we change this to `users`?

**Review by tmock12 — CHANGES_REQUESTED** [review]
Thanks! 

I never got around to adding all the endpoints as we didn't end up using the library. If you find it handy let me know and I can probably dedicate some time to working on it and adding more features.

File: lib/wordpress_ex/users.ex (hunk 1 above)
Comment (mikehostetler) [reply lib/wordpress_ex/users.ex]: Yep, my bad! Thanks for the catch

**Comment by mikehostetler** [comment]
I'm using it while porting a Wordpress site to Elixir/Phoenix. It's 80% learning exercise, but is slowly turning into something useful.

I'll throw ideas/comments/suggestions into issues as I come across them.  Thanks!

**Review by tmock12 — APPROVED** [review]

**Comment by tmock12** [comment]
Thanks @mikehostetler ! I'll add a few more endpoints on Friday and cut a minor version release. Until then feel free to use it from master (ie: `{:wordpress_ex, git: "https://github.com/tmock12/wordpress_ex"}`)

**Pull request merged**